Data/Graph/Dom.hs

```diff
@@ -614,4 +614,10 @@ fetch f i = do
 
 -- Redefine Data.Bifunctor.second for GHC 7 compatibility
 second :: (b -> c) -> (a, b) -> (a, c)
-second f (a, b) = (a, f b)
\ No newline at end of file
+second f (a, b) = (a, f b)
+
+#if __GLASGOW_HASKELL__ >= 704
```

Comment:
Try `#if MIN_VERSION_containers(0, 5, 0)`

Reply:
I'd actually revert that commit when we decide on dropping GHC 7.0.4 support.

Reply:
Fine!